.github/workflows/xpack-filebeat-macos-unit-tests.yml

```diff
@@ -0,0 +1,72 @@
+name: xpack-filebeat-macos-unit-tests
+
+env:
+  BEAT: "x-pack/filebeat"
+
+on:
+  pull_request:
+    paths:
+      - filebeat/
+      - x-pack/filebeat/
+      - x-pack/libbeat/
+      # OSS
+      - go.mod
+      - pytest.ini
+      - dev-tools/
+      - libbeat/**
+      # Workflow
+      - .github/workflows/xpack-filebeat-macos-unit-tests.yml
+      - .github/actions/unit-test-composite-action/action.yml
```

Comment:
This is not the current name:
- https://github.com/elastic/beats/tree/main/.github/actions/unit-test